### ISSUE TITLE
Training on custom data - where to put summary ?

### ISSUE THREAD

**Issue by GraphGrailAi** [nlpyang/PreSumm#128]
I want to implement abstractive training with `train_from=model_step_148000.pt` but where to put article and it's summary?

Should i process 1-5 steps noted in readme.md (from Option 2: process the data yourself) or not?

In issue #79  it is noted that i can simply put summary after `@highlight` in `raw_data/temp.raw_src` but there is no `@highlight` there.
https://github.com/nlpyang/PreSumm/issues/79

**Comment by binhna** [comment]
> I want to implement abstractive training with `train_from=model_step_148000.pt` but where to put article and it's summary?
> 
> Should i process 1-5 steps noted in readme.md (from Option 2: process the data yourself) or not?
> 
> In issue #79 it is noted that i can simply put summary after `@highlight` in `raw_data/temp.raw_src` but there is no `@highlight` there.
> #79

Yes, you need to follow the steps in README. And yes, one line of summary followed by one line of @highlight line. Here is an example:
![image](https://user-images.githubusercontent.com/18049173/74798613-cb0db500-5300-11ea-8097-07e6c9f95b0c.png)

**Comment by AyeshaSarwar** [comment]
Every @highlight means a separate summary? 

For our custom data, should we put @highlight for every line in our summary?

I am a little confused. Please if you could help me a little.
